src/tests/unit/tests/electron/device-connect-view/components/__snapshots__/electron-external-link.test.tsx.snap

```diff
@@ -2,8 +2,15 @@
 
 exports[`ElectronExternalLinkTest render 1`] = `
 <StyledLinkBase
+  0="t"
```

Comment:
This doesn't look correct. Can you please check this again?

Reply:
@lisli1 this snapshot tells me we were a little of on how we use the `children` props. You'd probably want to update `ElectronExternalLink` to render something like this:
```typescript
return <Link onClick={onClick}>{props.children}</Link>;
```
(before, we were passing `{...props.children}` inside the first `Link` tag... the `...` is the responsible for this odd-looking snapshot.